### DISCUSSION TITLE
脚本生成的.build目录疑问

### DISCUSSION THREAD

**Discussion by L-Super** [CefView/QCefView#276] [General]
`.sh` `.bat`生成的都是`.build/linux.x86_64`目录，即build目录在Linux下会是隐藏文件，想问一下在`build`前加`.`的作用是？

收到好友反馈，按照[doc](https://cefview.github.io/QCefView/zh/docs/intros/buiding/)方法进行编译出错，发现是路径不对，是`.build`不是`build`
@tishion

**Comment by tishion** [comment]
这个是个人喜好，没有有什么特别的地方，主要是用于区代码库中的目录。

**Reply by L-Super** [reply]
那我同步一下文档还是代码？现在文档和脚本代码有差异

**Reply by tishion** [reply]
文档是没问题的，你是不是看错了。

**Reply by L-Super** [reply]
> 文档是没问题的，你是不是看错了。

文档的代码执行，build没有.

**Reply by tishion** [reply]
你改改文档就行
